service_crategen/services.json

```diff
@@ -0,0 +1,170 @@
+{
```

Comment:
Thought for the future - this default config could all be generated by walking the botocore services directory and parsing out the most recent date.